### PR TITLE
[BEAM-1925] Remove deprecated context param from DoFn

### PR DESCRIPTION
**Pull request by sb2nov**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

R: @chamikaramj @robertwb PTAL

### CODE REVIEW

**Comment by sb2nov** [comment]
Travis seems happy for Python so should be safe to review

File: sdks/python/apache_beam/runners/common.py [hunk 1]
```diff
@@ -159,19 +157,15 @@ def invoke_start_bundle(self):
...
-    args = [self.context if d == core.DoFn.ContextParam else d
-            for d in defaults]
     self.output_processor.start_bundle_outputs(
-        self.signature.start_bundle_method.method_value(*args))
+        self.signature.start_bundle_method.method_value(*defaults))
 
   def invoke_finish_bundle(self):
     """Invokes the DoFn.finish_bundle() method.
     """
     defaults = self.signature.finish_bundle_method.defaults
```
Comment (chamikaramj) [inline sdks/python/apache_beam/runners/common.py]: Nit: rename variable to args_for_finish_bundle

File: sdks/python/apache_beam/runners/common.py [hunk 2]
```diff
@@ -159,19 +157,15 @@ def invoke_start_bundle(self):
     """Invokes the DoFn.start_bundle() method.
     """
     defaults = self.signature.start_bundle_method.defaults
```
Comment (sb2nov) [inline sdks/python/apache_beam/runners/common.py]: Done

File: sdks/python/apache_beam/runners/common.py (hunk 1 above)
Comment (sb2nov) [inline sdks/python/apache_beam/runners/common.py]: Done

**Comment by chamikaramj** [comment]
LGTM (lemme know if I should merge)

**Comment by sb2nov** [comment]
Please merge

**Comment by sb2nov** [comment]
@robertwb PTAL

**Comment by chamikaramj** [comment]
I'll wait for Robert's LGTM.

**Comment by robertwb** [comment]
LGTM, are there any docs we should be updating?

**Comment by sb2nov** [comment]
I checked the beam-site and the only mentions of ContextParam seem to be inside the pydoc.

**Comment by chamikaramj** [comment]
Sg. Merging.

**Pull request closed without merging**